src/diffusers/schedulers/scheduling_ddim.py

```diff
@@ -206,6 +207,7 @@ def step(
         model_output: torch.FloatTensor,
         timestep: int,
         sample: torch.FloatTensor,
+        prediction_type: str = "epsilon",
```

Comment:
Think this should go in the `__init__` function and we've somewhat settled on `predict_epsilon: bool` I think in terms of naming :-)

Reply:
Ah ok now we actually have three types, so we might have to reconsider this choice :sweat_smile: 

But I think it should definitely go in the config of the scheduler and not be an arg of `__call__`

Reply:
I actually messaged Nathan about my progress on DDIM v prediction in a separate branch as you commented - crazy timing!

I'll make sure I make these changes in my branch before opening a PR